src/RoadRegistry.Editor.Schema/Migrations/20210106144916_AddProjectionStatesErrorMessage.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace RoadRegistry.Editor.Schema.Migrations
+{
+    public partial class AddProjectionStatesErrorMessage : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "ErrorMessage",
+                schema: "RoadRegistryEditorMeta",
+                table: "ProjectionStates",
+                type: "nvarchar(max)",
+                nullable: true);
+
+            migrationBuilder.CreateTable(
```

Comment:
Hm ... have to investigate why this is happening.

Reply:
I'll wait to merge until this is figured out